test/webrick/test_httprequest.rb

```diff
@@ -123,7 +123,7 @@ def test_parse_header2()
       POST /foo/bar/../baz?q=a HTTP/1.0
       Content-Length: 9
       User-Agent:
-        FOO   BAR
+        FOO BAR
```

Comment:
Isn't it better to change the corresponding assertion to ensure that spaces are not squished?